.github/workflows/push_docker_image.yml

```diff
@@ -0,0 +1,23 @@
+name: push docker image
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: login
+        run: docker login --username '${{ secrets.DOCKER_USERNAME }}' --password '${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}'
+      - name: build
+        run: docker build . -t planetariumhq/ninechronicles-headless:git-${{ github.sha }} --build-arg COMMIT=git-${{ github.sha }} -t planetariumhq/ninechronicles-headless:latest --build-arg COMMIT=git-${{ github.sha }}
```

Comment:
Though this comment is not related to the above comment, does it need to pass the same build argument twice? 🤔

Reply:
@moreal It does appear alittle redundant 😅. We could adjust the Dockerfile so it doesn't require a version suffix as a build argument but I'll bring this issue up tomorrow!